handler/graceful-shutdown.md

```diff
@@ -0,0 +1,3 @@
+# graceful shutdown
```

Comment:
we should put it in doc folder instead of handler folder.

Reply:
oops, wrong click...
fixed